deeplabcut/utils/auxfun_videos.py

```diff
@@ -490,7 +493,7 @@ def DownSampleVideo(
     Downsamples the video to a width of 220 and height of 320 and saves it in C:\\yourusername\\rig-95\\Videos as reachingvideo1cropped.avi
     """
     writer = VideoWriter(vname)
-    return writer.rescale(width, height, rotateccw, outsuffix, outpath)
+    return writer.rescale(width, height, rotateccw, angle, outsuffix, outpath)
```

Comment:
Do the tests under tests/test_video.py all pass? 😊

Reply:
This is harder for me to answer. I run the file in ipython and nothing happened. I mean it didn't show any error but I'm not sure how this test file works and what should happen. I'm still only learning python

Reply:
Okay, I have a problem running pytest test_video.py. I'm getting:

```
  File "C:\Users\kdani\AppData\Roaming\Python\Python37\site-packages\path-2.2-py3.7.egg\path.py", line 886
    def mkdir(self, mode=0777):
                            ^
SyntaxError: invalid token
```
If I understand correctly it's because of the 0 in 0777 but I'm not sure how to correct that error

Reply:
Run the test, all 20 passed, no errors.

Reply:
Awesome! Thank you so much!